src/resources/postcss/tribe-ui.pcss

```diff
@@ -0,0 +1,140 @@
+:root {
+	--text-light-color: #777;
+	--text-dark-color: #464646;
+	--border-color: #dfdfdf;
+	--table-background-color: #f8f8f8;
+	--button-light-color: #009FF1;
+	--button-dark-color: #0080DB;
+}
+
+.tribe-button {
+	border: 0;
+	border-radius: 3px;
+	background: var(--button-light-color) linear-gradient( to bottom, var(--button-light-color), var(--button-dark-color) );
+	cursor: pointer;
+	color: #fff;
+	display: inline-block;
+	font-size: 14px;
+	font-weight: 400;
+	font-family: sans-serif;
+	letter-spacing: 1px;
+	line-height: 1;
+	height: auto;
+	padding: 10px;
+	text-align: center;
+	transition: all .15s ease-in;
+	-webkit-font-smoothing: subpixel-antialiased;
+
+	&:hover,
+	&:active,
+	&:focus {
+		background: var(--button-dark-color);
+	}
+}
+
+
+/* = Tickets & RSVP Modules Front End Styles
```

Comment:
Ticket and RSVP styles don't feel like they belong here - they'll add extra (needless) weight to sites that aren't running Event Tickets. I'd prefer to see them in Event Tickets.